server/auth/gatekeeper.go

```diff
@@ -286,7 +286,12 @@ func (s *gatekeeper) getClientsForServiceAccount(ctx context.Context, claims *ty
 	if err != nil {
 		return nil, err
 	}
-	_, clients, err := s.clientForAuthorization(authorization)
+	var clients *servertypes.Clients
+	if s.restConfig != nil {
+		_, clients, err = s.clientForAuthorization(authorization, s.restConfig.QPS, s.restConfig.Burst)
+	} else {
+		_, clients, err = s.clientForAuthorization(authorization, 5, 10)
```

Comment:
is 5, 10 lower than existing default?

Reply:
I print the default value，it is 0. In kubernetes, if the config's burst and qps valus is 0, it will be changed to 10,5.

Reply:
can you leave a comment with a permalink to the source of that? that way we know where it was taken from and where to find the default if it is updated in the future.

though ideally we'd fallback to the default implicitly instead of needing to copy it ourselves.